app/api/uploadURL.ts

```diff
@@ -0,0 +1,32 @@
+import { Storage } from '@google-cloud/storage'
+import { BlitzApiHandler } from 'blitz'
+
+import {
+  gcpStorageBucketName,
+  gcpStorageClientEmail,
+  gcpStoragePrivateKey,
+  gcpStorageProjectId,
+} from 'app/core/environment'
+
+const handler: BlitzApiHandler = async (req, res) => {
```

Comment:
This can be converted into a mutation, so you won't have to handle types yourself.